src/pages/request-assign/RequestRegisteredPage.tsx

```diff
@@ -1,19 +1,63 @@
 import ProfileHeader from '../../components/registeredPage/ProfileHeader';
 import PostTitleApply from '../../components/registeredPage/PostTitleApply';
 import AssignmentDescription from '../../components/registeredPage/AssignmentDescription';
+import RecruitInfo from '../../components/registeredPage/RecruitInfo';
 import QnASection from '../../components/postdetail/QnASection';
 import AuthorInfo from '../../components/postdetail/AuthorInfo';
 import styles from './requestRegisteredPage.module.scss';
 
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@tanstack/react-query';
+import { RequsetDetail } from '../../types/detailPageType';
+import { getRequestDetail } from '../../apis/detailPageAPI';
+
 export const RequestRegisteredPage = () => {
+  const { taskId } = useParams<{ taskId: string }>();
+  //const validIdeaId = ideaId ? Number(taskId) : undefined;
+
+  const { data, isLoading, error } = useQuery<RequsetDetail, Error>({
```

Comment:
data 이름을 직접 지정해주면 한 페이지에서 여러 요청 시에 조금 더 직관적인 코드가 됩니닷!

```suggestion
  const { data: requestData, isLoading, error } = useQuery<RequsetDetail, Error>({
```